azure-pipelines.yml

```diff
@@ -43,6 +43,35 @@ jobs:
           $(EXTRA_CMAKE_ARGS)
           $(Build.SourcesDirectory)
 
+    - script: |
+        set -e
+
+        # List all .cpp files to format, skipping all the ones inside the libraries folder, sort them and save the list on a file
+        find . -type d -path ./libraries -prune -o -type f -name "*.cpp" -print \
+        | sort > $(Build.BinariesDirectory)/files_to_format.txt
+
+        # Read back the list of files, modify them appending two newlines, so that the format_check.py is triggered
+        while read line; do echo -e "\n\n" >> $line; done <<< `cat $(Build.BinariesDirectory)/files_to_format.txt`
+
+        # Run the format check, save the command output and invert the exit code, since finding formatting issues is what we want
+        ! cmake --build . --target format_check > $(Build.BinariesDirectory)/format_output.txt
+
+        # Print the command output previously saved
+        cat format_output.txt
```

Comment:
This needs the build directory

Reply:
Right, copy paste error ftw!
I'll fix it in the script.